lib/views/pack/pack_view.dart

```diff
@@ -93,76 +85,26 @@ class _PackViewState extends ConsumerState<PackView>
     );
   }
 
-  Container _buildListTile(
-    BuildContext context,
-    String? title,
-    String? subtitle,
-    String type,
+  Widget _buildListTile(
+    PackItemsModel item,
     bool isLast,
-    bool? isCompletedTrack,
   ) {
-    var bodyLarge = Theme.of(context).primaryTextTheme.bodyLarge;
-    var hasSubtitle = subtitle.isNotNullAndNotEmpty();
-
-    return Container(
-      width: MediaQuery.of(context).size.width,
-      decoration: BoxDecoration(
-        border: isLast
-            ? null
-            : Border(
-                bottom: BorderSide(width: 0.9, color: ColorConstants.softGrey),
-              ),
-      ),
-      padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 20),
-      child: Row(
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-        crossAxisAlignment: CrossAxisAlignment.center,
-        children: [
-          Flexible(
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              mainAxisSize: MainAxisSize.min,
-              children: [
-                if (title.isNotNullAndNotEmpty())
-                  Text(
-                    title!,
-                    style: bodyLarge?.copyWith(
-                      color: ColorConstants.walterWhite,
-                      fontFamily: DmSans,
-                    ),
-                  ),
-                if (hasSubtitle)
-                  Flexible(
-                    child: Padding(
-                      padding: const EdgeInsets.only(top: 4),
-                      child: Text(
-                        subtitle!,
-                        style: bodyLarge?.copyWith(
-                          fontFamily: DmMono,
-                          color: ColorConstants.newGrey,
-                        ),
-                      ),
-                    ),
-                  ),
-              ],
-            ),
-          ),
-          _getIcon(type, isCompletedTrack: isCompletedTrack),
-        ],
-      ),
-    );
-  }
-
-  Widget _getIcon(String type, {bool? isCompletedTrack}) {
-    if (type == TypeConstants.LINK) {
-      return SvgPicture.asset(AssetConstants.icLink);
-    } else if (type == TypeConstants.TRACK &&
-        isCompletedTrack != null &&
-        isCompletedTrack) {
-      return Icon(Icons.check_circle_outline_rounded);
+    //ignore: prefer-conditional-expressions
+    if (item.type == TypeConstants.TRACK &&
+        item.isCompleted != null &&
+        !item.isCompleted!) {
```

Comment:
Can you not just do 
`if (item.type == TypeConstants.TRACK && item.isCompleted == false)`
?